edx-platform/pearson-studio-theme/cms/templates/index.html

```diff
@@ -195,9 +195,17 @@ <h2 class="title title-3">${_("Organization and Library Settings")}</h2>
               <label class="field-label">${_("Show all courses in organization:")}
                 % if org_names_list:
                 <select class="form-select" name="org">
-                  <option value="" selected></option>
-                  %for org in org_names_list:
-                    <option value="${org}">${org}</option>
+                  %if request.GET.get("org", ""):
+                    <option value="" selected></option>
+                  %else:
+                    <option value=""></option>
+                  %endif
```

Comment:
What do you think about using an inline if and adding a "All" message?. @elvinquero
```suggestion
                  <option value="" {%if request.GET.get("org") %} selected {%endif%}>All</option>
```

Reply:
Oh, I knew how to do inline in DjangoTemplate engine but I tried to do it in Mako and failed. I did not know I could use it like this, just like the DjangoTemplate syntax. @Jacatove